charts/pega/README.md

```diff
@@ -357,14 +370,25 @@ Use the `pegasearch` section to configure a deployment of ElasticSearch for sear
 Parameter   | Description   | Default value
 ---         | ---           | ---
 `image`   | Set the `pegasearch.image` location to a registry that can access the Pega search Docker image. The image is [available on DockerHub](https://hub.docker.com/r/pegasystems/search), and you may choose to mirror it in a private Docker repository. | `pegasystems/search:latest`
+`replicas` | Specify the desired replica count. | `1`
+`minimumMasterNodes` | To prevent data loss, it is vital to configure the minimumMasterNodes setting so that each master-eligible node knows the minimum number of master-eligible nodes that must be visible in order to form a cluster. This value should be configured using formula (n/2) + 1 where n is replica count or desired capacity.  See the ElasticSearch [important setting documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/important-settings.html) for more information. | `1`
```

Comment:
`minimumMasterNodes` | To prevent data loss, it is vital to configure the minimumMasterNodes setting so that each master-eligible node is set to the minimum number of master-eligible nodes that must be visible in order to form a cluster. Configure this value using the formula (n/2) + 1 where n is replica count or desired capacity.  For more information, see the ElasticSearch [important setting documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/important-settings.html) for more information. | `1`

Reply:
Updated